tiktokpy/models/feed.py

```diff
@@ -102,12 +127,11 @@ class FeedItem(BaseModel):
     music: MusicInfo
     stats: StatisticsInfo
     video: VideoInfo
+    image_post: ImagePostlInfo = None
```

Comment:
Optional?

Reply:
Sorry late response. I made it optional as it is not a property that is returned when it is a standard video.

Reply:
I now understand your comment.
Which way would you prefer?

Reply:
image_post: Optional[ImagePostlInfo]

Reply:
Thank you @Gr3gnov, I can not test it properly right now, I will try to push it tomorrow.

Reply:
Oh! And I will probably rename that class to ImagePostInfo.